docs/finality-provider-phase2.md

```diff
@@ -0,0 +1,589 @@
+`create-finality-provider` command.
+
+If successful, the command will return a transaction hash:
+
+``` shell
+{ "tx_hash":
+"C08377CF289DF0DC5FA462E6409ADCB65A3492C22A112C58EA449F4DC544A3B1" } 
+```
+
+ You can query this hash to confirm the transaction was successful by navigating
+ to the babylon chain and making a query, such as below:
+
+```shell 
+babylond query tx <transaction-hash> --chain-id bbn-test-5 
+```
+
+>Note: This query must be executed using the Babylon daemon (`babylond`), not
+the finality provider daemon (`fpd`), as the registration transaction is
+recorded on the Babylon blockchain.
+
+The hash returned should look something similar to below:
+
+```shell
+  type: message
+- attributes:
+  - index: true
+    key: fp value:
+  '{
+    "addr":"bbn1ht2nxa6hlyl89m8xpdde9xsj40n0sxd2f9shsq",
+    "description":
+    {
+      "moniker":"MyFinalityProvider",
+      "identity":"
+      ","website":"https://myfinalityprovider.com",
+      "security_contact":"security@myfinalityprovider.com",
+      "details":"Reliablefinality provider for the Babylon
+        network"
+    },
+      "commission":"0.050000000000000000",
+      "btc_pk":"cf0f03b9ee2d4a0f27240e2d8b8c8ef609e24358b2eb3cfd89ae4e4f472e1a41",
+      "pop":{"btc_sig_type":"BIP340",
+      "btc_sig":"YJgc6NU7Z011imqSfPc9w/Namr1hFj48oTlEjGqbAVvHJv+9h3p/1shTohEb1g0fDWij7Ti9yKZzjAgNVepObA=="},
+      "slashed_babylon_height":"0",
+      "slashed_btc_height":"0",
+      "jailed":false,
+      "consumer_id":"euphrates-0.5.0"
+    }'
+  - index: true
+    key: msg_index value: "0"
+  type: babylon.btcstaking.v1.EventNewFinalityProvider
+gas_used: "82063" gas_wanted: "94429" height: "66693" info: "" logs: [] raw_log:""
+```
+
+When a finality provider is created, it's associated with two key elements:
+
+**a) BTC Public Key:** - This serves as the unique identifier for the finality
+provider.  - It's derived from a Bitcoin private key, likely using the secp256k1
+elliptic curve.  - This key is used in the Bitcoin-based security model of
+Babylon.
+
+**b) Babylon Account:** - This is an account on the Babylon blockchain.  - It's
+where staking rewards for the finality provider are sent.  - This account is
+controlled by the key you use to create and manage the finality provider (the
+one you added with fpd keys add).
+
+This dual association allows the finality provider to interact with both the
+Bitcoin network (for security) and the Babylon network (for rewards and
+governance).
+
+## Slashing Conditions
+
+Slashing occurs when a finality provider **double signs**. This occurs when a
+finality provider signs conflicting blocks at the same height. This results in
+the extraction of the provider's private key and automatically triggers shutdown
+of the finality provider.
+
+### Withdrawing Rewards
+
+When withdrawing rewards, you need to use the Babylon chain's CLI since rewards
+are managed by the main chain.
```

Comment:
We would have a cmd within `fpd` after addressing https://github.com/babylonlabs-io/finality-provider/issues/13.